package.json

```diff
@@ -45,7 +45,7 @@
     "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-jest": "^27.6.0",
     "husky": "^8.0.3",
-    "jest": "^29.7.0",
+    "jest": "^30.0.0-alpha.1",
```

Comment:
Apparently inline snapshots aren't compatible with `prettier@3` before this version :shrug: (https://github.com/jestjs/jest/issues/14305#issuecomment-1785221918)